release-team/role-handbooks/test-infra/README.md

```diff
@@ -1,4 +1,5 @@
-# Test Infra Lead Handbook
+# Test Infra Lead Handbook 
+Note: This role was deprecated after the 1.15 release.  Starting with 1.16 this role was mostly merged with [Branch Management](https://github.com/sig-release/blob/updating-test-infra-role-references/release-team/role-handbooks/branch-manager/README.md) 
```

Comment:
The responsibliities were split between branch manager and test-infra-oncall. 
Hence we should indicate that all the maintenance and test-infra related questions should be directed to test-infra-oncall at the channels #testing-ops #sig-testing.

Reply:
> ... maintenance and test-infra related questions should be directed to test-infra-oncall at the channels #testing-ops #sig-testing.

Adding this to the branch manager handbook as well.